### PR TITLE
Create Success screen UI

### PR DESCRIPTION
**Pull request by kacper-echaust**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,16 +1,12 @@
-import { useState } from 'react';
 import './App.css';
 import { SplashScreen } from './Components/SplashScreen/SplashScreen';
-import { Form } from './Components/Form/Form';
+import {  PaniniProvider } from './context/PaniniContext';
```
Comment (NerdbordMentor) [inline src/App.jsx:3]: **suggestion**: Consider removing the extra space in the import statement for consistency.

File: src/App.jsx [hunk 2]
```diff
@@ -1,16 +1,12 @@
...
 
 function App() {
-  const [animate, setAnimate] = useState(false);
-
   return (
-    <>
-      <SplashScreen animate={animate} setAnimate={setAnimate} />
-      {animate && <Form animate={animate} />}
-    </>
+    <PaniniProvider>
```
Comment (NerdbordMentor) [inline src/App.jsx:7]: **praise**: Good use of context provider to wrap the application components.

File: src/Components/Form/Form.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useContext } from 'react';
```
Comment (NerdbordMentor) [inline src/Components/Form/Form.tsx:1]: **suggestion**: Importing `useContext` is a good move for managing state with context.

File: src/Components/Form/Form.tsx [hunk 4]
```diff
@@ -16,11 +16,17 @@ import { Section } from './Section/Section';
...
-  animate: boolean;
-};
+import { PaniniContext, SplashScreenValues } from '../../context/PaniniContext';
 
-const Form = ({ animate }: Props) => {
+const Form = () => {
+  const { animate, setAnimate, setSplashScreenValues } = useContext(PaniniContext);
+  const sendOrder = () => {
+    setAnimate(false);
+    setSplashScreenValues({
```
Comment (NerdbordMentor) [inline src/Components/Form/Form.tsx:25]: **issue**: Ensure that the `sendOrder` function handles any potential errors or edge cases when setting state.

File: src/Components/SplashScreen/SplashScreen.tsx [hunk 5]
```diff
@@ -1,12 +1,11 @@
-import React from 'react';
+import React, { useContext } from 'react';
```
Comment (NerdbordMentor) [inline src/Components/SplashScreen/SplashScreen.tsx:1]: **suggestion**: Importing `useContext` here is a good choice for accessing context values.

File: src/Components/SplashScreen/SplashScreen.tsx [hunk 6]
```diff
@@ -1,12 +1,11 @@
...
+import { Form } from '../Form/Form';
 
-type Props = {
-  animate: boolean;
-  setAnimate: React.Dispatch<React.SetStateAction<boolean>>;
-};
 
-const SplashScreen = ({ animate, setAnimate }: Props) => {
+const SplashScreen = () => {
+  const { animate, setAnimate,splashScreenValues } = useContext(PaniniContext);
```
Comment (NerdbordMentor) [inline src/Components/SplashScreen/SplashScreen.tsx:8]: **suggestion**: Transitioning to use context for `animate` and `setAnimate` improves state management.

File: src/context/PaniniContext.tsx [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  setSplashScreenValues: () => {},
+});
+
+const PaniniProvider = ({ children }: Props) => {
+  const [animate, setAnimate] = useState(false);
+  const [splashScreenValues, setSplashScreenValues] = useState({
+    headerName: SplashScreenValues.PaniniCreator,
+    buttonName: SplashScreenValues.Begin,
+  });
+console.log(splashScreenValues);
```
Comment (NerdbordMentor) [inline src/context/PaniniContext.tsx:39]: **nitpick**: Consider removing the `console.log` statement to clean up the code before production.

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Overall Feedback

The code changes introduce a new context, `PaniniContext`, which effectively manages the state for animations and splash screen values. The refactoring of the `App`, `Form`, and `SplashScreen` components to use this context is well-executed, leading to a cleaner and more maintainable codebase. The `sendOrder` function in the `Form` component correctly updates the context state, allowing the splash screen to display the appropriate messages after form submission. However, the task requirements specifically mention the creation of a Success screen form section based on a provided design, which is not addressed in the code changes. The current implementation focuses on state management and context usage but lacks the actual UI components for the success screen.

### Things Missing:

- [ ] Success screen form section based on the provided design.

### Things That Went Well:

- The use of context for state management is well-implemented, reducing prop drilling and improving code readability.
- The refactoring of components to consume context values is efficient and aligns with modern React practices.

**Review by Sumick — APPROVED** [review]

**Pull request merged**